trustfall_core/test_data/tests/valid_queries/fold_with_no_inner_output_with_count_output.graphql.ron

```diff
@@ -0,0 +1,28 @@
+TestGraphQLQuery (
+    schema_name: "numbers",
+    // This @fold statement doesn't actually need to fully evaluate:
+    // it has no outputs, and the predicate can be determined to be satisfied
+    // with only a partial evaluation of the @fold.
+    //
+    // The number 30 is composite and has 3 prime factors: 2, 3, 5.
+    // This test checks whether the evaluation of the @fold is stopped early
+    // after two of those three primes are produced (satisfying the predicate).
```

Comment:
I think this is not true — in fact the opposite is true. The `@output` on the count requires us to fully evaluate the `@fold`, or else we don't know what the count is.

Did you mean to put this comment in a different file?

Reply:
Yeah, it's from an earlier test that I copied, not meant to be here after I added the `@output`. I've fixed it :)